### ISSUE TITLE
How long to train and evaluate on SBD?

### ISSUE THREAD

**Issue by LetsGoFir** [saic-vul/fbrs_interactive_segmentation#3]
It takes me 10 hours to do the evaluation on SBD, is it normal?

**Comment by ptrvilya** [comment]
It depends on the model that you are using and your hardware, but overall this time seems to be too large. ResNet34 on one 1080ti takes about 10 minutes for one training epoch and 1 minute for one validation epoch.

**Comment by LetsGoFir** [comment]
> It depends on the model that you are using and your hardware, but overall this time seems to be too large. ResNet34 on one 1080ti takes about 10 minutes for one training epoch and 1 minute for one validation epoch.

I use the 'resnet50_dh128_sbd' and my gpu is rtx2080, so the time is not normal, right?
The SBD has 6671 items to do the inference, and each one takes about 10s.

**Comment by ptrvilya** [comment]
Yes, this time is not normal. The normal time for ResNet50 on SBD is about 15 minutes per training epoch on one 1080ti with batch-size=12

**Comment by LetsGoFir** [comment]
I download the SBD from 
http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz
and type this command
`python3 scripts/evaluate_model.py RGB-BRS --checkpoint=resnet50_dh128_sbd --datasets=SBD`
I think the number 6671 is right for SBD evaluation, let me check it.

**Comment by ptrvilya** [comment]
I thought you were talking about training and evaluation during training. Evaluating the NoC metric for RGB-BRS takes a long time because every image is processed sequentially click by click. So your timing seems normal :)

**Comment by XavierCHEN34** [comment]
hello, I enquire how long would it take to train on COCO + LVIS on 4 GPUs? The training setting is the same as that of SBD?
